guides/common/modules/proc_exporting-the-library-environment-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,63 @@
+[id="Exporting_the_Library_Environment_in_a_Syncable_Format_{context}"]
+= Exporting the Library Environment in a Syncable Format
+
+You can export contents of all Yum repositories in the Library environment of an organization to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
```

Comment:
Is `Yum` correct here? the list in line 26-28 also contains ks and file repos.

Reply:
@parthaa Can you confirm?

Reply:
Yes Yum, KS and file repos